Tests/DatadogTests/Datadog/LoggerTests.swift

```diff
@@ -562,7 +562,7 @@ class LoggerTests: XCTestCase {
 
         logMatchers.forEach {
             $0.assertValue(
-                forKeyPath: RUMContextAttributes.IDs.sessionID,
+                forKeyPath: "application_id",
```

Comment:
This is currently enforcing the schema used by logs backend. 
I'm wondering if it would be preferable to use the output of the mapping function `mapInternalAttributeKey` here instead of the actual expected key and leave that enforce to E2E tests.